src/Rooms.ts

```diff
@@ -143,3 +143,19 @@ function guessDMRoomTargetId(room: Room, myUserId: string): string {
     if (oldestUser === undefined) return myUserId;
     return oldestUser.userId;
 }
+
+export const UNSTABLE_MSC2867_MARKED_UNREAD_TYPE = 'com.famedly.marked_unread';
```

Comment:
we should be using the js-sdk's `UnstableValue` class.

Reply:
done within https://github.com/matrix-org/matrix-react-sdk/pull/6314/commits/6dcd393181a5b605355d306912914fe45eb27050